[BUG] Etcd client init failed when using etcd cluster

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Using etcd to sync data, Etcd client init failed when using multiple etcd instance like : `http://localhost:2379,http://localhost:2380,http://localhost:2381` .


### Expected Behavior

_No response_

### Steps To Reproduce

1. edit application.yml like this
![image](https://user-images.githubusercontent.com/62384022/143256928-adaee699-0318-47b6-a764-720ca0cfb304.png)
2. start admin failed
3. start bootstrap failed


### Environment

```markdown
ShenYu version(s): latest in master
```


### Debug logs

1. admin: 
```
2021-11-24 22:24:35 [grpc-default-executor-0] WARN  io.etcd.jetcd.resolver.SmartNameResolver - Error wile getting list of servers
io.etcd.jetcd.common.exception.EtcdException: Unable to resolve endpoints [http://localhost:2379,http://localhost:2380,http://localhost:2381]
	at io.etcd.jetcd.common.exception.EtcdExceptionFactory.newEtcdException(EtcdExceptionFactory.java:30)
	at io.etcd.jetcd.resolver.SmartNameResolver.doResolve(SmartNameResolver.java:137)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2021-11-24 22:24:35 [grpc-default-executor-0] WARN  io.grpc.internal.ManagedChannelImpl - [Channel<1>: (etcd)] Failed to resolve name. status=Status{code=NOT_FOUND, description=null, cause=null}
2021-11-24 22:24:35 [main] ERROR org.apache.shenyu.admin.listener.etcd.EtcdClient - check node exists error.
java.util.concurrent.ExecutionException: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1908)
	at org.apache.shenyu.admin.listener.etcd.EtcdClient.exists(EtcdClient.java:62)
	at org.apache.shenyu.admin.listener.etcd.EtcdDataInit.run(EtcdDataInit.java:48)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:784)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:768)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.shenyu.admin.ShenyuAdminBootstrap.main(ShenyuAdminBootstrap.java:36)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:588)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:567)
	at io.etcd.jetcd.ClientConnectionManager.lambda$execute$4(ClientConnectionManager.java:331)
	at net.jodah.failsafe.Functions.lambda$toCtxSupplier$15(Functions.java:268)
	at net.jodah.failsafe.Functions.lambda$getPromise$1(Functions.java:80)
	at net.jodah.failsafe.RetryPolicyExecutor$1.call(RetryPolicyExecutor.java:101)
	at net.jodah.failsafe.RetryPolicyExecutor.lambda$supplyAsync$1(RetryPolicyExecutor.java:139)
	at net.jodah.failsafe.Functions.lambda$null$3(Functions.java:97)
	at net.jodah.failsafe.internal.util.DelegatingScheduler.lambda$schedule$0(DelegatingScheduler.java:141)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.grpc.StatusRuntimeException: NOT_FOUND
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:497)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.etcd.jetcd.ClientConnectionManager$AuthTokenInterceptor$1$1.onClose(ClientConnectionManager.java:289)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:426)
	at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:689)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$900(ClientCallImpl.java:577)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:751)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:740)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	... 3 common frames omitted
2021-11-24 22:24:35 [main] INFO  org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener - 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2021-11-24 22:24:35 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
java.lang.IllegalStateException: Failed to execute CommandLineRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:787)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:768)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.shenyu.admin.ShenyuAdminBootstrap.main(ShenyuAdminBootstrap.java:36)
Caused by: org.apache.shenyu.common.exception.ShenyuException: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
	at org.apache.shenyu.admin.listener.etcd.EtcdClient.exists(EtcdClient.java:66)
	at org.apache.shenyu.admin.listener.etcd.EtcdDataInit.run(EtcdDataInit.java:48)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:784)
	... 5 common frames omitted
```
2. bootstrap:
```
2021-11-24 22:24:55 [grpc-default-executor-0] WARN  io.etcd.jetcd.resolver.SmartNameResolver - Error wile getting list of servers
io.etcd.jetcd.common.exception.EtcdException: Unable to resolve endpoints [http://localhost:2379,http://localhost:2380,http://localhost:2381]
	at io.etcd.jetcd.common.exception.EtcdExceptionFactory.newEtcdException(EtcdExceptionFactory.java:30)
	at io.etcd.jetcd.resolver.SmartNameResolver.doResolve(SmartNameResolver.java:137)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2021-11-24 22:24:55 [grpc-default-executor-0] WARN  io.grpc.internal.ManagedChannelImpl - [Channel<1>: (etcd)] Failed to resolve name. status=Status{code=NOT_FOUND, description=null, cause=null}
2021-11-24 22:24:55 [main] ERROR org.apache.shenyu.sync.data.etcd.EtcdSyncDataService - java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
java.util.concurrent.ExecutionException: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1908)
	at org.apache.shenyu.sync.data.etcd.EtcdClient.getChildrenKeys(EtcdClient.java:104)
	at org.apache.shenyu.sync.data.etcd.EtcdSyncDataService.etcdClientGetChildren(EtcdSyncDataService.java:319)
	at org.apache.shenyu.sync.data.etcd.EtcdSyncDataService.watcherData(EtcdSyncDataService.java:88)
	at org.apache.shenyu.sync.data.etcd.EtcdSyncDataService.<init>(EtcdSyncDataService.java:81)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration.syncDataService(EtcdSyncDataConfiguration.java:66)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052.CGLIB$syncDataService$0(<generated>)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052$$FastClassBySpringCGLIB$$866da08c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052.syncDataService(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:651)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:636)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.refresh(ReactiveWebServerApplicationContext.java:66)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.shenyu.bootstrap.ShenyuBootstrapApplication.main(ShenyuBootstrapApplication.java:35)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: NOT_FOUND
	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:588)
	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:567)
	at io.etcd.jetcd.ClientConnectionManager.lambda$execute$4(ClientConnectionManager.java:331)
	at net.jodah.failsafe.Functions.lambda$toCtxSupplier$15(Functions.java:268)
	at net.jodah.failsafe.Functions.lambda$getPromise$1(Functions.java:80)
	at net.jodah.failsafe.RetryPolicyExecutor$1.call(RetryPolicyExecutor.java:101)
	at net.jodah.failsafe.RetryPolicyExecutor.lambda$supplyAsync$1(RetryPolicyExecutor.java:139)
	at net.jodah.failsafe.Functions.lambda$null$3(Functions.java:97)
	at net.jodah.failsafe.internal.util.DelegatingScheduler.lambda$schedule$0(DelegatingScheduler.java:141)
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266)
	at java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.grpc.StatusRuntimeException: NOT_FOUND
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:497)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.etcd.jetcd.ClientConnectionManager$AuthTokenInterceptor$1$1.onClose(ClientConnectionManager.java:289)
	at io.grpc.internal.DelayedClientCall$DelayedListener$3.run(DelayedClientCall.java:464)
	at io.grpc.internal.DelayedClientCall$DelayedListener.delayOrExecute(DelayedClientCall.java:428)
	at io.grpc.internal.DelayedClientCall$DelayedListener.onClose(DelayedClientCall.java:461)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:617)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:70)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:803)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:782)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	... 3 common frames omitted
2021-11-24 22:24:55 [main] WARN  org.springframework.boot.web.reactive.context.AnnotationConfigReactiveWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'syncDataService' defined in class path resource [org/apache/shenyu/springboot/sync/data/etcd/EtcdSyncDataConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.sync.data.api.SyncDataService]: Factory method 'syncDataService' threw exception; nested exception is java.lang.IllegalStateException: Stream is already completed, no further calls are allowed
2021-11-24 22:24:55 [grpc-default-executor-1] ERROR io.grpc.internal.SerializingExecutor - Exception while executing runnable io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed@3636e100
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@75cae55e rejected from java.util.concurrent.ScheduledThreadPoolExecutor@2ceea9f2[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
	at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2063)
	at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:830)
	at java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:326)
	at java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:533)
	at com.google.common.util.concurrent.MoreExecutors$ScheduledListeningDecorator.schedule(MoreExecutors.java:643)
	at io.etcd.jetcd.WatchImpl$WatcherImpl.onError(WatchImpl.java:265)
	at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:449)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.etcd.jetcd.ClientConnectionManager$AuthTokenInterceptor$1$1.onClose(ClientConnectionManager.java:289)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:617)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:70)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:803)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:782)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2021-11-24 22:24:55 [main] INFO  org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener - 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2021-11-24 22:24:55 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'syncDataService' defined in class path resource [org/apache/shenyu/springboot/sync/data/etcd/EtcdSyncDataConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.sync.data.api.SyncDataService]: Factory method 'syncDataService' threw exception; nested exception is java.lang.IllegalStateException: Stream is already completed, no further calls are allowed
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:656)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:636)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.refresh(ReactiveWebServerApplicationContext.java:66)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.apache.shenyu.bootstrap.ShenyuBootstrapApplication.main(ShenyuBootstrapApplication.java:35)
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.shenyu.sync.data.api.SyncDataService]: Factory method 'syncDataService' threw exception; nested exception is java.lang.IllegalStateException: Stream is already completed, no further calls are allowed
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:651)
	... 19 common frames omitted
Caused by: java.lang.IllegalStateException: Stream is already completed, no further calls are allowed
	at com.google.common.base.Preconditions.checkState(Preconditions.java:502)
	at io.grpc.stub.ClientCalls$CallToStreamObserverAdapter.onNext(ClientCalls.java:345)
	at io.etcd.jetcd.WatchImpl$WatcherImpl.resume(WatchImpl.java:151)
	at io.etcd.jetcd.WatchImpl.watch(WatchImpl.java:81)
	at org.apache.shenyu.sync.data.etcd.EtcdClient.watchChildChange(EtcdClient.java:147)
	at org.apache.shenyu.sync.data.etcd.EtcdSyncDataService.watcherData(EtcdSyncDataService.java:93)
	at org.apache.shenyu.sync.data.etcd.EtcdSyncDataService.<init>(EtcdSyncDataService.java:81)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration.syncDataService(EtcdSyncDataConfiguration.java:66)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052.CGLIB$syncDataService$0(<generated>)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052$$FastClassBySpringCGLIB$$866da08c.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.apache.shenyu.springboot.sync.data.etcd.EtcdSyncDataConfiguration$$EnhancerBySpringCGLIB$$ecf7d052.syncDataService(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 20 common frames omitted
```


### Anything else?

_No response_

--- a/shenyu-admin/src/main/java/org/apache/shenyu/admin/config/DataSyncConfiguration.java
+++ b/shenyu-admin/src/main/java/org/apache/shenyu/admin/config/DataSyncConfiguration.java
@@ -189,7 +189,7 @@ public class DataSyncConfiguration {
         @Bean
         public EtcdClient etcdClient(final EtcdProperties etcdProperties) {
             Client client = Client.builder()
-                    .endpoints(etcdProperties.getUrl())
+                    .endpoints(etcdProperties.getUrl().split(","))
                     .build();
             return new EtcdClient(client);
         }

--- a/shenyu-register-center/shenyu-register-server/shenyu-register-server-etcd/src/main/java/org/apache/shenyu/register/server/etcd/client/EtcdClient.java
+++ b/shenyu-register-center/shenyu-register-server/shenyu-register-server-etcd/src/main/java/org/apache/shenyu/register/server/etcd/client/EtcdClient.java
@@ -71,7 +71,7 @@ public class EtcdClient {
                         .setDaemon(true)
                         .build());
 
-        this.client = Client.builder().endpoints(urls).build();
+        this.client = Client.builder().endpoints(urls.split(",")).build();
 
         try {
             initLease();

--- a/shenyu-spring-boot-starter/shenyu-spring-boot-starter-sync-data-center/shenyu-spring-boot-starter-sync-data-etcd/src/main/java/org/apache/shenyu/springboot/sync/data/etcd/EtcdSyncDataConfiguration.java
+++ b/shenyu-spring-boot-starter/shenyu-spring-boot-starter-sync-data-center/shenyu-spring-boot-starter-sync-data-etcd/src/main/java/org/apache/shenyu/springboot/sync/data/etcd/EtcdSyncDataConfiguration.java
@@ -75,7 +75,7 @@ public class EtcdSyncDataConfiguration {
     @Bean
     public EtcdClient etcdClient(final EtcdConfig etcdConfig) {
         Client client = Client.builder()
-                .endpoints(etcdConfig.getUrl())
+                .endpoints(etcdConfig.getUrl().split(","))
                 .build();
         return new EtcdClient(client);
     }

